roles/core/log_client/readme.rst

```diff
@@ -34,13 +34,62 @@ Log client verbosity defaults to info, it can be one of the following (defined i
 | debug    | debug-level messages             |
 +----------+----------------------------------+
 
+You can force usage of a static file content instead of a generated one by 
+setting **log_client_static_configuration** into inventory.
+**log_client_static_configuration** is a multi lines variable that should contain
+the full desired content of the target rsyslog.conf file.
+
+For example:
+
+.. code-block:: yaml
+
+  log_client_static_configuration: |
+    # rsyslog configuration file
+
+    # For more information see /usr/share/doc/rsyslog-*/rsyslog_conf.html
+    # If you experience problems, see http://www.rsyslog.com/doc/troubleshoot.html
+
+    #### MODULES ####
+
+    # The imjournal module bellow is now used as a message source instead of imuxsock.
+    $ModLoad imuxsock # provides support for local system logging (e.g. via logger command)
+    $ModLoad imjournal # provides access to the systemd journal
+    #$ModLoad imklog # reads kernel messages (the same are read from journald)
+    #$ModLoad immark  # provides --MARK-- message capability
+    ...
+
+| You can also split the configuration into several files, using **log_client_custom_conf**. 
```

Comment:
What is the purpose of '|' at the beginning of each lines?

Also, if the variable for static configuration is **log_client_static_configuration**, I would name this one **log_client_custom_configuration**. That might also avoid conflict if one wants to write a custom.conf file.

Reply:
Each "|" is used to start a newline, the default indentation did not feel right, should I remove them ?

Reply:
OK I got it. It looks weird when reading the readme in the terminal. If this is the same paragraph, I don't see no reason to force the newline, so please remove them.